Author: Andreas Metzler <ametzler@debian.org>
Description: Fix GNU/Hurd detection
 Comparing "gcc -dM -E - < /dev/null | sort -u " on GNU/Hurd and Linux seems
 to tell me that GNU/Hurd can be deteced with #ifdef __gnu_hurd__.
 OTOH "HURD" is not defined. If you'd rather have a upper case macro
 #if defined __GNU__ && defined __MACH__
 should also work.
Bug-Debian: http://bugs.debian.org/629142

--- autogen-5.16.2.orig/agen5/fmemopen.c
+++ autogen-5.16.2/agen5/fmemopen.c
@@ -15,7 +15,7 @@ typedef struct {
     size_t     eof;
 } fmemc_get_buf_addr_t;
 
-#ifdef HURD
+#ifdef __gnu_hurd__
 #define _IOT__IOTBASE_fmemc_get_buf_addr_t sizeof(fmemc_get_buf_addr_t)
 #endif
 
